docs/blog/2020-05-26-introducing-will-it-build/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Introducing Will It Build: A new builds benchmarking service for Gatsby Cloud"
+date: 2020-05-26
+author: Shannon Soper
+excerpt: "The first -- and only -- publicly available static site benchmarking service, Will It Build demonstrates the progress of build times for sample Gatsby benchmark sites on Gatsby Cloud. Thanks to our new Incremental Builds feature, data and content changes now go from edit to live in seconds! See for yourself: all of our data can be queried right on the site using our exposed GraphQL API at willit.build/api-playground."
+tags:
+  - willitbuild
+  - gatsby-cloud
+  - incremental-builds
+  - performance
+---
+
+## The first -- and only -- publicly available static site benchmarking service
+
+Today we are thrilled to announce that [Will It Build](https://willit.build/) is live! This benchmarking website lets you view and compare build times for Gatsby-built websites and applications on Gatsby Cloud.
+
+Will It Build demonstrates the progress of build times for sample Gatsby benchmark sites on Gatsby Cloud. We prototyped some generic versions of the many types of sites that can be built with Gatsby, from basic developer blog to large social network with an image on every page, using multiple popular CMS's and content sources. Currently we are showing results for Wordpress, Contentful, Drupal, and DatoCMS as well as sites using MDX and Markdown as the data source. (We will be adding more, so contact [partners@gatsbyjs.com](mailto:partners@gatsbyjs.com) to get the guide for how to create a benchmark for your CMS or service!)
+
+For those curious as to how it all works, we have open sourced [the example benchmark sites](https://github.com/gatsbyjs/gatsby/tree/master/benchmarks) as well as [the benchmark plugin](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-benchmark-reporting) that reports back the results. Additionally, all of our data can be queried right on the site using our exposed GraphQL API at [willit.build/api-playground](https://willit.build/api-playground).
+
+One of the coolest things about WIllit.build is how clearly it shows the drastic reduction in build times that our new [Incremental Builds](https://www.gatsbyjs.org/blog/2020-04-22-announcing-incremental-builds/) feature makes possible on Gatsby Cloud:
+
+![](https://lh3.googleusercontent.com/9bzilTs1JiTr_VGOuDktdNZ4G-GxBKxot1nX4fdXngUo-CDVrQMSmdaUMnJM7w_Loj6-ncxuv71ebaC5h0_g56tSL6tP6P8P2OxWyA_SbczAFMnc0LLgQkdw35ffuEFvfYUAfk3p)
```

Comment:
Alt text needed here :)

Reply:
good catch! this is what happens when i start work before coffee